Makefile

```
@@ -37,8 +36,20 @@ else # otherwise linux
 	docker run --privileged -e DISPLAY=${DISPLAY} -d -v /tmp/.X11-unix:/tmp/.X11-unix -v $(abspath $(VERACRUZ_ROOT)):/work/veracruz -v $(HOME)/.cargo/registry/:/usr/local/cargo/registry/ --name  $(VERACRUZ_CONTAINER) $(VERACRUZ_DOCKER_IMAGE)_tz
 endif
 
+.PHONY:
+nitro: build
+ifeq ($(OS_NAME),darwin)
+	docker run --privileged -e DISPLAY=$(IP):0 -d -v /tmp/.X11-unix:/tmp/.X11-unix -v $(abspath $(VERACRUZ_ROOT)):/work/veracruz -v $(HOME)/.cargo/registry/:/usr/local/cargo/registry/ --name  $(VERACRUZ_CONTAINER) $(VERACRUZ_DOCKER_IMAGE)_nitro
+else # otherwise linux
+	docker run --privileged -e DISPLAY=${DISPLAY} -d -v /tmp/.X11-unix:/tmp/.X11-unix -v $(abspath $(VERACRUZ_ROOT)):/work/veracruz -v $(HOME)/.cargo/registry/:/usr/local/cargo/registry/ --name  $(VERACRUZ_CONTAINER) $(VERACRUZ_DOCKER_IMAGE)_nitro
+endif
+
 .PHONY:
 build: Dockerfile
+ifeq ($(TEE),nitro)
+	git clone https://github.com/aws/aws-nitro-enclaves-cli.git
```

We might want to fail silently in case the repo already exists.
`git clone https://github.com/aws/aws-nitro-enclaves-cli.git || echo "failure"` for instance?

---

I've been thinking about what to do with this. Currently if it already exists, it errors out (not silently).
Ideally, it would not error out if it exists, but instead pull updates, and only rebuild if there are changes. 

That's a bit more than I want to bite off right now, but yes, something should be done.